test/cql/cdc_too_short_stream_id_test.cql

```diff
@@ -0,0 +1,7 @@
+create table tb (pk int primary key) with cdc = {'enabled': true};
+insert into tb (pk) VALUES (0);
+
+-- Key of length != 128 b should return empty result set (issue #6570)
+select * from tb_scylla_cdc_log where "cdc$stream_id" = 0x00;
+
+select * from tb_scylla_cdc_log where "cdc$stream_id" = 0x;
```

Comment:
@kostja this is an interesting example of the limitations of the pure-CQL tests without any programming language.
hey insert one item into the table, and will get one entry in the CDC log. But they don't know which key this entry will have - it's a random key. Even if we can seed it the same and get the same on every run, it will be very annoying if we change this random function in the future and random tests start to fail.
So all this test can test is the empty select case (where a bad key doesn't match anything). They can't check a successful case because 1. they don't know what "where" to use, and 2. if they knew, the result will contain the random key and can change.